tests/communication/test_tes_connection.py

```diff
@@ -10,7 +10,7 @@
 from tes_client.communication.request_sender import RequestSender
 from tes_client.communication.tes_connection import TesConnection
 
-__TES_ENDPOINT = 'inproc://TES'
+__TES_ENDPOINT = 'inproc://Omega'
 __REQUEST_SENDER_ENDPOINT = 'inproc://REQUEST_SENDER'
 __RESPONSE_RECEIVER_ENDPOINT = 'inproc://RESPONSE_RECEIVER'
 __TES_SOCKET_IDENTITY = b'TES_SOCKET'
```

Comment:
[Suggestion]
It also should be changed.
Also repository itself should be renamed

Reply:
Renames of the repository, classes, functions, and variables will be in a separate pull request.